ui/helpers/constants/design-system.ts

```diff
@@ -55,6 +55,8 @@ export enum BackgroundColor {
   backgroundDefault = 'background-default',
   backgroundAlternative = 'background-alternative',
   backgroundMuted = 'background-muted',
+  backgroundSection = 'background-section',
+  backgroundSubsection = 'background-subsection',
```

Comment:
Adding new background colors section and subsection to design system BackgroundColor enum